ui/raidboss/data/04-sb/raid/o6s.js

```diff
@@ -52,7 +53,7 @@
             en: 'Demonic Stone on YOU',
             de: 'Dämonischer Stein auf DIR',
             fr: 'Pierre démoniaque sur VOUS',
-            ko: '악령의 돌 장판 → 나',
+            ko: '악령의 돌 장판 대상자',
```

Comment:
I don't know Korean, but it looks like you're changing many (but not all) of the responses with arrows to something else.  I think if this is something you want to change, it would be better to follow up and change all arrow functions in its own pull request rather than modifying some of them here.  I think it's important to be consistent in trigger text.

 It looks like these were added by @Bing-su a year ago.  @Bing-su, do you have any thoughts about changing all of these?

Reply:
Recently instance like e8s, '--- on YOU' translations are like my requests. `on YOU`, `→ 나` and `대상자` are same meaning, but `대상자` is tune....I think.
I changed all Arrow function that I found in this request. and I thought it was small enough to call it an update..

Reply:
I was very rude for bing-su.. I'll contact and ask thoughts him right now.

Reply:
It's very easy for me to look at a patch that is only adding new lines and think "this is clearly an improvement over non-existing translations".  When there are modifications mixed in too, then I start asking more questions like "what do other translators think about these lines being changed?" and "is this change consistent with other translations?" and "has this PR changed everything or are there other things that need to be changed too?"

Reply:
Sorry to confuse you. I'll undo arrow functions and request again when I'm done talking to other translators!

Reply:
@Una-P 제가 옛날에 번역할땐 텍스트로 쓴게 그대로 TTS로 나온다는걸 몰랐었죠... 그래서 화살표를 썼습니다. 저는 요즘 파판에 흥미가 떨어져서 여기도 잘 안오게 되더라구요. 만약 번역하다가 제가 작업한걸 덮어씌우는게 낫다 싶으시면 제 허락 필요없이 덮어씌우셔도 괜찮습니다.

Reply:
@Bing-su 허락 감사합니다.